paddle/platform/enforce.h

```diff
@@ -0,0 +1,116 @@
+/*
```

Comment:
The copyright format is not right, please use  [this](https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/gserver/layers/AddtoLayer.h#L1)